theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
IIRC correctly that yellow was used for notification popups in AdCentre, we referred to it as "Scott's favourite colour" owing to his bitterness at needing to use it.  It probably is used elsewhere but we probably should get away from it.